keybase/platform_windows.go

```diff
@@ -191,12 +199,13 @@ func (c context) Apply(update updater.Update, options updater.UpdateOptions, tmp
 		return fmt.Errorf("No asset")
 	}
 	var args []string
-	auto, _ := c.config.GetUpdateAuto()
-	canBeSilent, _ := CheckCanBeSilent(update.Asset.LocalPath, c.log, CheckRegistryUninstallCode)
-	if auto && canBeSilent {
+	_, auto := c.config.GetUpdateAuto()
+	if auto && !c.config.GetUpdateAutoOverride() {
+
 		args = append(args, "/quiet")
 	}
-	_, err := command.Exec(update.Asset.LocalPath, args, time.Hour, c.log)
+	// We really don't want to kill installers, hence 30 days here
+	_, err := command.Exec(update.Asset.LocalPath, args, time.Hour*24*30, c.log)
```

Comment:
The timeout is meant to prevent a bad installer from preventing updates so... 

I think maybe this should be max a few days?

What happens again if you kill the installer when it's prompting?